x-pack/plugins/apm/ftr_e2e/cypress/integration/power_user/settings/custom_links.spec.ts

```diff
@@ -36,7 +36,11 @@ describe('Custom links', () => {
   });
 
   it('clears filter values when field is selected', () => {
+    cy.intercept('GET', '/internal/apm/settings/custom_links?*').as(
+      'customLinks'
+    );
     cy.visitKibana(basePath);
+    cy.wait('@customLinks');
     cy.contains('Create custom link').click();
```

Comment:
The button is clicked before the page has finished loading. This can cause the button to be removed before it can be clicked.